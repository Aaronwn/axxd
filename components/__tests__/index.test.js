const warnSpy = jest.spyOn(console, 'warn').mockImplementation(() => {});
const axxd = require('..');

describe('axxd', () => {
  it('exports modules correctly', () => {
    expect(Object.keys(axxd)).toMatchSnapshot();
  });

  it('should hint when import all components', () => {
    expect(warnSpy).toBeCalledWith(
      'You are using a whole package of axxd, please use https://www.npmjs.com/package/babel-plugin-import to reduce app bundle size.'
    );
    warnSpy.mockRestore();
  });
});
