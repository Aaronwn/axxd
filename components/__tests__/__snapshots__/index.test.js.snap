// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`antd exports modules correctly 1`] = `
Array [
  "Divider",
  "version",
]
`;
